File: solutions/round-1/solutions/bugtrons-1.cs
// C# code for Max
// Water Container

using System;
class bugtrons1
{

public static int maxArea(int[] m)
{

    int Area = 0;
    
    for(int i = 0; i < m.Length; i++)
    {
        for(int j = i + 1; j < m.Length; j++)
        {
            Area = Math.Max(Area, Math.Min(m[i], m[j]) *
                            (j - i));
        }
    }
    return Area;
}

// Driver code
public static void Main(String[] args)
{
    int []a = { 2, 6, 5, 4 };
    int []b = { 4, 2, 3, 5, 6 };

    Console.WriteLine(maxArea(a));
    Console.Write(maxArea(b));
}
}

File: solutions/round-1/solutions/bugtrons-1.py
# Python3 code for Maximum Water Container

def maxArea(M, Length) :
    area = 0
    for i in range(Length) :
        for j in range(i + 1, Length) :
           
            # Calculating the max area
            area = max(area, min(M[j], M[i]) * (j - i))

    return area
 
# Driver code
a = [ 2, 6, 5, 4 ]
b = [ 4, 2, 3, 5, 6 ]
 
len1 = len(a)
print(maxArea(a, len1))
 
len2 = len(b)
print(maxArea(b, len2))
